visualization.spectrum/resources/language/resource.language.en_us/strings.po

```diff
@@ -95,3 +95,23 @@ msgstr "%i°"
 msgctxt "#30019"
 msgid "Turn continuously"
 msgstr "Turn continuously"
+
+msgctxt "#30020"
+msgid "Tilt"
+msgstr "Tilt"
+
+msgctxt "#30021"
+msgid "Base"
+msgstr "Base"
+
+msgctxt "#30022"
+msgid "%+i%%"
+msgstr "%+i%%"
```

Comment:
What makes the `+` before the `i`, not know as this, makes it only then number if higher as 0?

Reply:
Was just kinda cosmetics because actually "0%" is 100% (default) and "+10%" is 110% and so on.
Removed now as i don't know how to do this with the new format.